### PR TITLE
Added Detailed CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by say-het**
Thanks @aslams2020 for assigning me this issue.

 - Fixed Issue #49 
 
Please merge this PR

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Contribution Guidelines
+
+- **New Features**: Ensure new features are well-documented and add to the overall functionality of the project.
+- **Code Quality**: Keep code modular, and adhere to project coding standards.
+- **Commit Messages**: Use clear and concise commit messages.
+- **Documentation**: Update relevant documentation for new features or significant changes.
+- **Testing**: Ensure changes are thoroughly tested, especially responsive design elements.
+
+## Code of Conduct
```
Comment (aslams2020) [inline CONTRIBUTING.md:80]: Add the link to CODE OF CONDUCT here, from the repo

**Comment by aslams2020** [comment]
Add the link to CODE OF CONDUCT , from the repo

**Comment by say-het** [comment]
Hey @aslams2020 I have fixed it you can merge now!

**Review by aslams2020 — APPROVED** [review]
Approved

**Pull request merged**